### PR TITLE
San Juan

### PR DESCRIPTION
**Pull request by andrewmassengale**
Here's the San Juan rulebook.

### CODE REVIEW

File: rulebooks/san-juan.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+**The player with the most Victory Points is the winner.** A tie is won by the player with the most cards in his hand and goods.
+
+# Roles
+
+Some buildings enhance these actions and privileges.
+
+## Councillor
+
+**Action**
```
Comment (camden) [inline rulebooks/san-juan.md]: Can we make these actions / privileges level 3 headers? (###)

**Comment by camden** [comment]
Hey Andrew! Looks good! Just needs some minor fixes. Also, feel free to add your rulebook to the "Recently Added" section of `config/homepage.yml` while you're at it. :)

File: rulebooks/san-juan.md (hunk 1 above)
Comment (andrewmassengale) [reply rulebooks/san-juan.md]: Done.

**Comment by camden** [comment]
Thanks @andrewmassengale! :)

**Pull request merged**